activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -124,6 +124,8 @@ def self.extract_value_from_default(default)
             # Numeric types
             when /\A\(?(-?\d+(\.\d*)?\)?)\z/
               $1
+            when /\A'(-?\d+(\.\d*)?\)?)'::integer\z/i
+              $1
```

Comment:
This particular patch is needed for Postgres 10.5+. Without this an ActiveRecord test fails so it felt like the right place to include this patch was in activerecord itself rather than in a patch in rails2_ruby2.

Reply:
let's move this to a `rails2_ruby2` patch